web/src/components/Link.tsx

```diff
@@ -0,0 +1,12 @@
+export function Link({ href, linkText }: { href: string; linkText: string }) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
```

Comment:
Great with a Link component! We could use it in several other places, but that would probably also require more customization and stuff we don't really need now :)

On thing here though, I'm a bit unsure on the `noreferrer` attribute here, it basically means that the "receiver" will not see where the user was coming from. This also means that if we use this component for the links to our commercial website, all traffic will show as "direct" instead of "coming from the app".

Reply:
I get this error when I give rel as a prop. So I think we need it to open the page in a new tab.

![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/95029552/fb452ce1-f390-46d8-9990-10e3ed9a15a3)

Reply:
Let's update the `.eslintrc.json` with this then: 
```json
    "react/jsx-no-target-blank": [
      "error",
      {
        "allowReferrer": true
      }
    ],
```